lib/_http_outgoing.js

```diff
@@ -697,11 +688,9 @@ function connectionCorkNT(conn) {
 
 OutgoingMessage.prototype.addTrailers = function addTrailers(headers) {
   this._trailer = '';
-  const keys = ObjectKeys(headers);
   const isArray = ArrayIsArray(headers);
   var field, value;
-  for (var i = 0, l = keys.length; i < l; i++) {
-    var key = keys[i];
+  for (const key of ObjectKeys(headers)) {
```

Comment:
We should not make this change for performance reasons.

Reply:
resolved in https://github.com/nodejs/node/pull/30958/commits/17049d56621869d8776a922d5e56b6c5ec35be7b